sdb/stocksdashboard.py

```diff
@@ -227,21 +227,20 @@ def plot_stock(self, input_data=None, p=None, column='adj_close',
             p.add_tools(hover)
         return p
 
-    def build_dashboard(self, data1=None, data2=None, params={}, params2={},
-                        title="stocks.py example", **kwargs):
-        if params and not params2:
-            params2 = params
-
-        p1 = self.plot_stock(input_data=data1, params=params, **kwargs)
-        p2 = self.plot_stock(input_data=data2, params=params2, **kwargs)
-
-        # open a browser
-        layout = gridplot([p1, p2],
-                          plot_width=self.width, plot_height=self.height,
+    def build_dashboard(self,
+                        data_list=[],
+                        params_list=[],
+                        title="stocks.py example"):
+        plots = []
+        for data, params in zip(data_list, params_list):
```

Comment:
I have to check this out, but we can't have infinite vertical plots due to the limitation of the screen.